README.md

```diff
@@ -1,33 +1,81 @@
+- Submit a pull request. During the pull request phase, the REP will undergo design feedback from both Bicep core maintainers and the broader community. The author should be prepared to make revisions based on this feedback.
+- Build consensus and integrate feedback.
+- When the Bicep core maintainers feel confident about the design and have reached a consensus, they should proceed to vote on whether to accept or reject the REP.
+
+### Active
+
+Once a REP is reviewed and approved, it is considered active. The author should take the following steps to maintain the active state of the REP:
+- Replace `0000` in the REP file name with the PR number.
+_ Merge the PR so that the REP file will appear in the active folder in the main branch.
+- Proceed with the implementation of the proposed feature.
+- Optionally, consider putting the feature behind a preview feature flag. This would give the author the opportunity to easily make revisions and breaking changes.
+- As the author works towards a final implementation, submit PRs to the bicep-reps repository to keep the active REP in sync with the implementation.
+
+### Final
+
+REPs in the Final state are considered fully complete and implemented without any preview feature flag. Any proposed changes should be made through a new REP which enhances or deprecates the final REP. To convert an active REP to a final, follow these steps:
+- Ensure that the active REP is fully implemented and well tested, and there will be no breaking changes to the customer facing API contract.
+- Submit a PR to move the REP from the active folder to the final folder.
+    - Optionally, as part of the PR change, if the REP marked as final enhances an existing final REP, include in the existing REP a `Enhanced By` field to reference the new REP.
+
+### Inactive
+
+An active REP should be moved inactive if the feature does not receive enough positive user feedback. To do so, the author should send a PR to move the REP file from active to inactive.
+
+### Deprecated
+
+A final REP should marked as deprecated if a new REP improves it in a non-backward compatible way. Follow the steps below:
+- When the new REP reaches the final state, as part of the PR change, move the old REP to the deprecated folder.
+- Within the deprecated REP file, add a `Deprecated By` attribute and reference the new REP.
+
+### Rejected
+
+A rejected REP will lead to a closed PR. In practice, this occurrence should be rare.
+
+### Bicep REP Lifecycle Visualization
+
+Refer to the state diagram below for a visual representation of the Bicep REP lifecycle.
+
+```mermaid
+stateDiagram-v2
+    state Review <<choice>>
+    [*] --> Draft: Open a PR
+    Draft --> Review: Review the PR
+    Review --> Active: PR accepted
+    Review --> Rejected: PR rejected
+    Active --> Active: Create new PRs to\n make revision
```

Comment:
Do we need an Experimental REP state similar to how PowerShell RFCs handle experimental features?

Reply:
My expectation is that most REP features are likely to be initially implemented as experimental, so I'm not sure if it's worth maintaining two active states, one for non-experimental features and one for experimental features. What do you think?